tools/cockpit.spec

```diff
@@ -158,7 +158,9 @@ exec 2>&1
     --disable-silent-rules \
     --with-cockpit-user=cockpit-ws \
     --with-selinux-config-type=etc_t \
-    %{?rhel:--without-storaged-iscsi-sessions} \
+%if 0%{?rhel} < 8
```

Comment:
This condition is true for non-RHEL builds as well, no?  I.e., doesn't this include `--without-storaged-iscsi-sessions` in all builds except RHEL 8, including all Fedora builds?

Reply:
True that, thanks for spotting!

Reply:
Fixed